lib/l10n/app_es.arb

```diff
@@ -0,0 +1,539 @@
+  "nextButtonLabel": "SIGUIENTE",
+  "showButtonLabel": "MOSTRAR",
+  "hideButtonLabel": "OCULTAR",
+  "continueButtonLabel": "CONTINUAR",
+
+  "cancelTooltip": "Cancelar",
+  "changeTooltip": "Cambiar",
+  "clearTooltip": "Limpiar",
+  "previousTooltip": "Anterior",
+  "nextTooltip": "Siguiente",
+  "showTooltip": "Mostrar",
+  "hideTooltip": "Ocultar",
+  "actionRemove": "Remover",
+  "resetButtonTooltip": "Restablecer",
+
+  "doubleBackExitMessage": "Presione “atrás” nuevamente para salir.",
+
+  "sourceStateLoading": "Cargando",
+  "sourceStateCataloguing": "Catalogando",
+  "sourceStateLocatingCountries": "Ubicando países",
+  "sourceStateLocatingPlaces": "Ubicando lugares",
+
+  "chipActionDelete": "Borrar",
+  "chipActionGoToAlbumPage": "Mostrar en Álbumes",
+  "chipActionGoToCountryPage": "Mostrar en Países",
+  "chipActionGoToTagPage": "Mostrar en Etiquetas",
+  "chipActionHide": "Esconder",
+  "chipActionPin": "Fijar",
+  "chipActionUnpin": "Dejar de fijar",
+  "chipActionRename": "Renombrar",
+  "chipActionSetCover": "Elegir portada",
+  "chipActionCreateAlbum": "Crear álbum",
+
+  "entryActionCopyToClipboard": "Copiar al portapapeles",
+  "entryActionDelete": "Borrar",
+  "entryActionExport": "Exportar",
+  "entryActionInfo": "Información",
+  "entryActionRename": "Renombrar",
+  "entryActionRotateCCW": "Rotar en sentido antihorario",
+  "entryActionRotateCW": "Rotar en sentido horario",
+  "entryActionFlip": "Voltear horizontalmente",
+  "entryActionPrint": "Imprimir",
+  "entryActionShare": "Compartir",
+  "entryActionViewSource": "Ver fuente",
+  "entryActionViewMotionPhotoVideo": "Abrir foto en movimiento",
+  "entryActionEdit": "Editar con…",
+  "entryActionOpen": "Abrir con…",
+  "entryActionSetAs": "Establecer como…",
+  "entryActionOpenMap": "Mostrar en aplicación de mapa…",
+  "entryActionRotateScreen": "Rotar pantalla",
+  "entryActionAddFavourite": "Agregar a favoritos",
+  "entryActionRemoveFavourite": "Quitar de favoritos",
+
+  "videoActionCaptureFrame": "Capturar fotograma",
+  "videoActionPause": "Pausa",
+  "videoActionPlay": "Reproducir",
+  "videoActionReplay10": "Retroceder 10 segundos",
+  "videoActionSkip10": "Adelantar 10 segundos",
+  "videoActionSelectStreams": "Seleccionar pistas",
+  "videoActionSetSpeed": "Velocidad de reproducción",
+  "videoActionSettings": "Ajustes",
+
+  "entryInfoActionEditDate": "Editar fecha y hora",
+  "entryInfoActionEditRating": "Editar clasificación",
+  "entryInfoActionEditTags": "Editar etiquetas",
+  "entryInfoActionRemoveMetadata": "Eliminar metadatos",
+
+  "filterFavouriteLabel": "Favorito",
+  "filterLocationEmptyLabel": "No localizado",
+  "filterTagEmptyLabel": "Sin etiquetar",
+  "filterRatingUnratedLabel": "Sin clasificar",
+  "filterRatingRejectedLabel": "Rechazado",
+  "filterTypeAnimatedLabel": "Animado",
+  "filterTypeMotionPhotoLabel": "Foto en movimiento",
+  "filterTypePanoramaLabel": "Panorámica",
+  "filterTypeRawLabel": "Sin procesar",
```

Comment:
`Raw` is a [format](https://es.wikipedia.org/wiki/Raw_(formato)), and should be kept as `Raw`, without translation.